osquery/core/process.h

```diff
@@ -66,6 +66,10 @@ class PlatformProcess : private boost::noncopyable {
   explicit PlatformProcess() : id_(kInvalidPid) {}
   explicit PlatformProcess(PlatformPidType id);
 
+#ifdef WIN32
+  explicit PlatformProcess(pid_t pid);
```

Comment:
Mind adding a note, or docstring, to why `PlatformPidType` is not a `pid_t` on both platforms?